src/test/kotlin/uk/gov/justice/digital/hmpps/gradle/DpsSpringBootPluginTest.kt

```
@@ -13,64 +19,90 @@ import org.springframework.boot.gradle.plugin.SpringBootPlugin
 
 class DpsSpringBootPluginTest {
 
-    val project: Project = ProjectBuilder.builder().build()
+  val project: Project = ProjectBuilder.builder().build()
 
-    @BeforeEach
-    fun `Create project`() {
-        project.pluginManager.apply("uk.gov.justice.digital.hmpps.gradle.DpsSpringBoot")
-    }
+  @BeforeEach
+  fun `Create project`() {
+    project.pluginManager.apply("uk.gov.justice.digital.hmpps.gradle.DpsSpringBoot")
+  }
+
+  @Nested
+  inner class Plugins {
 
-    @Nested
-    inner class Plugins {
+    @Test
+    fun `Should apply the Spring Boot plugin`() {
+      val thrown = catchThrowable { project.plugins.getPlugin(SpringBootPlugin::class.java) }
```

why not just let the exception propagate instead i.e. 
```
project.plugins.getPlugin(SpringBootPlugin::class.java)
```

---

the exception will then be thrown by the test method and will fail

---

Yeah I thought about that, but it's not always obvious that a test without an assertion is actually testing the lack of an exception. Whereas the above is obvious.  But... :man_shrugging: I'll switch back to your way in the next PR.

---

might therefore add a comment above along the lines of
```
// will throw exception if plugin fails to apply
```